core/domain/state_services.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Commands that can be used to operate on states"""
```

Comment:
Add period at end.

That said, I think you can get rid of this file, and have a classmethod on the State object instead. Something like State.convert_state_dict_to_yaml(). Then use cls.from_dict(state_dict) within it.

Reply:
Done.